src/CoreWCF.RabbitMQ/tests/Helpers/MessageContainer.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+
+namespace CoreWCF.RabbitMQ.Tests.Helpers
+{
+    internal class MessageContainer
+    {
+        public static Stream GetTestMessage()
+        {
+            string currentDirectory = AppDomain.CurrentDomain.BaseDirectory ?? throw new Exception("AppDomain current directroy is empty");
+            string path = Path.Combine(currentDirectory, "Resources/rabbitmqTestMessage.bin");
```

Comment:
Does Environment.CurrentDirectory do the same thing? AppDomain is only there for code compat reasons and a few methods which need to be available somewhere (like type loaders etc) so I like to avoid it's usage.